spec/unit/modulesync_spec.rb

```diff
@@ -1,11 +1,13 @@
 require 'spec_helper'
 
+require 'modulesync/puppet_module'
```

Comment:
Why is this line needed? I'd have expected it to be there since `ModuleSync` is already required and that loads it.

Reply:
You're right, its not needed.

Reply:
Done.